cli/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "clean": "rimraf build .cache"
   },
-  "devDependencies": {
```

Comment:
Why did you move this devDependencies (only use on dev time i guess), to dependencies ?

Reply:
Because they are actually not dev deps be deps for the CLI to work if we were to release it.